src/world/area_kkj/kkj_10/ACAB20.c

```diff
@@ -7,6 +7,19 @@ static char* N(exit_str_3) = "kkj_19";
 static char* N(exit_str_4) = "kkj_20";
 static char* N(exit_str_5) = "kkj_29";
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_10/ACAB20", func_80240000_ACAB20);
+ApiStatus func_80240000_ACAB20(Evt* script, s32 isInitialCall) {
+    s32 i;
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_10/ACAB20", func_80240034_ACAB54);
+    for (i = 0; i < 7; i++) {
```

Comment:
Instead of `7` here, we tend to do ARRAY_COUNT(thing) instead - so `ARRAY_COUNT(D_80151328->sources)`